WordPress/Classes/Networking/Remote Objects/RemoteBlogSettings.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+
+
+/**
+ *  @class           RemoteBlogSettings
+ *  @brief           This class encapsulates all of the *remote* settings available for a Blog entity
+ */
+public class RemoteBlogSettings : NSObject
```

Comment:
(Just a minor note), Optionals don't map to ObjC. However... since the parsers deal with default values, it'll be safe to just move over to native data types. (On it already, yay!)

Thank you!